GameData/RealAntennas/Parts/NFE_CAE_RTREDEV.cfg

```diff
@@ -0,0 +1,245 @@
+ %MODULE[ModuleRealAntenna]{ %antennaDiameter = 6 }
+ }
+ 
+// C5+ RelayTech One
+ @PART[RTGigaDish2]:HAS[@MODULE[ModuleDataTransmitter]]:FOR[RealAntennas]
+{
+
+ !MODULE[ModuleDataTransmitter] {}
+ %MODULE[ModuleRealAntenna]{ %antennaDiameter = 5 }
+ }
+
+//D-2 Spot Antenna
+@PART[nfex-antenna-top-dish-1]:HAS[@MODULE[ModuleDataTransmitter]]:FOR[RealAntennas]
+{
+
+ !MODULE[ModuleDataTransmitter] {}
+ %MODULE[ModuleRealAntenna]{ %antennaDiameter = 0.25 }
+ }
+
+// D-50 Large Spot Antenna
+ @PART[nfex-antenna-top-dish-2]:HAS[@MODULE[ModuleDataTransmitter]]:FOR[RealAntennas]
+{
+
+ !MODULE[ModuleDataTransmitter] {}
+ %MODULE[ModuleRealAntenna]{ %antennaDiameter = 0.5 }
+ }
+
+// DR-1 High Gain Antenna
+ @PART[nfex-antenna-static-mini-1]:HAS[@MODULE[ModuleDataTransmitter]]:FOR[RealAntennas]
+{
+
+ !MODULE[ModuleDataTransmitter] {}
+ %MODULE[ModuleRealAntenna]{ %referenceGain = 3 }
+ }
+
+// RFL-100 Giant Dish Reflector
+ @PART[nfex-antenna-reflector-huge-1]:HAS[@MODULE[ModuleDeployableReflector]]:FOR[RealAntennas]
+{
+
+ !MODULE[ModuleDataTransmitter] {}
+ %MODULE[ModuleRealAntenna]{ %antennaDiameter = 14.9 }
+ }
+
+// RA-X3 Phased Relay Antenna
+ @PART[nfex-antenna-phased-array-3]:HAS[@MODULE[ModuleDataTransmitter]]:FOR[RealAntennas]
+{
+
+ !MODULE[ModuleDataTransmitter] {}
+ %MODULE[ModuleRealAntenna]{ %antennaDiameter = 0.944 }
+ }
+
+// RA-00-2 Micro-Relay Antenna
+ @PART[nfex-antenna-relay-tiny-1]:HAS[@MODULE[ModuleDataTransmitter]]:FOR[RealAntennas]
+{
+
+ !MODULE[ModuleDataTransmitter] {}
+ %MODULE[ModuleRealAntenna]{ %antennaDiameter = 0.3 }
+ }
+
+// F-DA Direct Antenna Feed
+ @PART[nfex-antenna-feeder-direct-1]:HAS[@MODULE[ModuleDataTransmitter]]:FOR[RealAntennas]
+{
+
+ !MODULE[ModuleDataTransmitter] {}
+ %MODULE[ModuleRealAntenna]{ %referenceGain = 2.5 }
+ }
+
+// F-RA Relay Antenna Feed
+ @PART[nfex-antenna-feeder-relay-1]:HAS[@MODULE[ModuleDataTransmitter]]:FOR[RealAntennas]
+{
+
+ !MODULE[ModuleDataTransmitter] {}
+ %MODULE[ModuleRealAntenna]{ %referenceGain = 3 }
+ }
+
+// PH-1 Phased Array Antenna Element
+ @PART[nfex-antenna-phased-single-1]:HAS[@MODULE[ModuleDataTransmitter]]:FOR[RealAntennas]
+{
+
+ !MODULE[ModuleDataTransmitter] {}
+ %MODULE[ModuleRealAntenna]{ %referenceGain = 2 }
+ }
+
+// PH-2 Phased Array Antenna Element
+ @PART[nfex-antenna-phased-single-2]:HAS[@MODULE[ModuleDataTransmitter]]:FOR[RealAntennas]
+{
+
+ !MODULE[ModuleDataTransmitter] {}
+ %MODULE[ModuleRealAntenna]{ %referenceGain = 2.5 }
+ }
+
+// PH-3 Phased Array Antenna Element
+ @PART[nfex-antenna-phased-single-3]:HAS[@MODULE[ModuleDataTransmitter]]:FOR[RealAntennas]
+{
+
+ !MODULE[ModuleDataTransmitter] {}
+ %MODULE[ModuleRealAntenna]{ %referenceGain = 2.5 }
+ }
+ 
+// CommTech EXP-VR-2T
+ @PART[RTLongAntenna3]:HAS[@MODULE[ModuleDataTransmitter]]:FOR[RealAntennas]
+{
+
+ !MODULE[ModuleDataTransmitter] {}
+ %MODULE[ModuleRealAntenna]{ %referenceGain = 3 }
+ }
+
+//-------------------------
+//Parts not converted
+//-------------------------
```

Comment:
Since you seem to have done this with an automated tool, can you:
Remove the extra blank line after the patch open {
Properly indent the next two patch lines
Ensure a closing newline at the end of each file